docs-user/guide-remote-profiling.md

```diff
@@ -1,28 +1,125 @@
+ - [The "reference browser"](https://github.com/mozilla-mobile/reference-browser)
+ - [Fenix](https://github.com/mozilla-mobile/fenix)
+ - [The GeckoView-example app](https://searchfox.org/mozilla-central/source/mobile/android/geckoview_example)
+
+Among these, Firefox for Android is the only one that has a Nightly channel which uses a recent [mozilla-central](https://hg.mozilla.org/mozilla-central/)-based Gecko, updates automatically (because it's on the Play Store), and is usable as a daily browser. However, it doesn't use GeckoView or e10s.
+
+Mozilla's current development efforts on mobile are focused on GeckoView. GeckoView has very different performance characteristics from non-GeckoView Gecko, mainly because GeckoView uses e10s: it runs web content in a different process from the browser process. Some new features, such as WebRender, are even exclusively targeting GeckoView. So if you do performance investigations and profiling, you'll really want to be profiling GeckoView, which means that Firefox for Android is not a good choice for a profiling target.
+
+That leaves the other four products:
+
+ - Focus is usable as a daily browser, and it has [a "Nightly" channel](https://github.com/mozilla-mobile/focus-android/wiki/Release-tracks#nightly), but that channel [uses GeckoView Beta](https://github.com/mozilla-mobile/focus-android/blob/master/build.gradle#L9-L10). This makes it almost useless for profiling work - you don't want to spend your time finding problems that have already been fixed. The Gecko Beta channel is also missing some recent profiling improvements.
+ - The reference browser uses a more recent version of GeckoView: It tries to stick very closely to mozilla-central, but since it uses [a pinned version of GeckoView](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt#L6), it only refreshes to a new build of GeckoView whenever somebody manually updates the pinned version. [This happens around once a week](https://github.com/mozilla-mobile/android-components/commits/master/buildSrc/src/main/java/Gecko.kt). The reference browser is somewhat usable as a regular browser. It is not on the Google Play Store, so it does not auto-update, but you can install it from an apk manually.
+ - Fenix is supposed to become a GeckoView-based replacement for Fennec in the future, but at this point, it is in a very early stage and there might not even be downloadable builds for it.
+ - Lastly, there's the GeckoView-example app: This app is an extremely simple GeckoView-based app, which is generated along with every GeckoView build on the mozilla-central trees. That means you can get GeckoView-example apks for any GeckoView build you want. GeckoView-example is also the easiest GeckoView-based app to compile locally. It's not really usable as your daily browser, and it's not on the Play Store so it doesn't auto-update, but it's usually enough for performance testing and profiling work.
```

Comment:
This is the best write-up I read about our current status on mobile, and I think this should be moved, or just duplicated, somewhere else (likely MDN ?).